ocs/ocp.py

```diff
@@ -204,3 +205,50 @@ def wait_for_resource(
                 return True
 
         return False
+
+    def exec_cmd_on_pod(self, pod_name, command):
+        """
+        Execute a command on a pod (e.g. oc rsh)
+
+        Args:
+            pod_name (str): The pod on which the command should be executed
+            command (str): The command to execute on the given pod
+
+        Returns:
+            Munch Obj: This object represents a returned yaml file
+        """
+        rsh_cmd = f"rsh {pod_name} "
+        rsh_cmd += command
+        return self.exec_oc_cmd(rsh_cmd)
+
+
+# the following functions location is temporary, will be changed to ocs class
+def get_ceph_tools_pod():
+    """
+    Get the Ceph tools pod
+
+    Returns:
+        str: The Ceph tools pod name
+    """
+    ocp_pod_obj = OCP(kind='pods', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    ct_pod = ocp_pod_obj.get(
+        selector='app=rook-ceph-tools'
+    ).toDict()['items'][0]['metadata']['name']
+    assert ct_pod, f"No Ceph tools pod found"
+    return ct_pod
+
+
+def exec_ceph_cmd(ceph_cmd):
+    """
+    Execute a Ceph command on the Ceph tools pod
+
+    Args:
+        ceph_cmd (str): The Ceph command to execute on the Ceph tools pod
+
+    Returns:
+        dict: Ceph command output
+    """
+    ocp_pod_obj = OCP(kind='pods', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    ct_pod = get_ceph_tools_pod()
+    ceph_cmd += " --format json-pretty"
+    return ocp_pod_obj.exec_cmd_on_pod(ct_pod, ceph_cmd).toDict()
```

Comment:
Not all ceph command line tools has `--format json-pretty` option. Eg. `rados` knows it, but eg. `rbd` doesn't. But I don't think we need to address it here, one would just run `exec_cmd_on_pod(get_ceph_tools_pod(), cmd)` when one needs something like that.